pkg/controllers/clusterresourceplacement/controller.go

```diff
@@ -222,12 +223,12 @@ func (r *Reconciler) handleUpdate(ctx context.Context, crp *fleetv1beta1.Cluster
 			klog.V(2).InfoS("Placement has finished the rollout process and reached the desired status", "clusterResourcePlacement", crpKObj, "generation", crp.Generation)
 			r.Recorder.Event(crp, corev1.EventTypeNormal, "PlacementRolloutCompleted", "Placement has finished the rollout process and reached the desired status")
 		}
-		metrics.FleetPlacementStatus.WithLabelValues(crp.Name).Set(1)
+		metrics.FleetPlacementComplete.WithLabelValues(crp.Name).SetToCurrentTime()
```

Comment:
i remember i never see people emit the cpu metrics with the timestamp. Could you please investigate how people create the gauge metrics?

Reply:
I talked to Arvind about this too as my reaction was exactly the same "I  never see people emit metrics with the timestamp" but it turned out there is actually a reason for this

Reply:
the timestamp is there to help to setup alerts for "snapshot" instead of including all the history